packages/dev-tools/server/DevToolsServer.js

```diff
@@ -52,7 +52,31 @@ export async function startAsync(projectDir) {
   const port = await freeportAsync(19002, { hostnames: [null, 'localhost'] });
   const server = express();
 
-  const authenticationContext = await createAuthenticationContextAsync({ port });
+  const devtoolsHost = () => {
+    let devtoolHost;
+    if (process.env.EXPO_DEVTOOLS_LISTEN_ADDRESS) {
+      devtoolHost = process.env.EXPO_DEVTOOLS_LISTEN_ADDRESS.trim();
+    } else {
+      devtoolHost = `localhost`;
+    }
+    return devtoolHost;
+  };
+  const listenHostname = devtoolsHost();
+
+  const devtoolsGraphQLHost = () => {
```

Comment:
i think we can move the `devtoolsGraphQLHost` and `devtoolsHost` functions definition outside of `startAsync`, this would make the code easier to read.

Reply:
Moved function definitions outside of `startAsync`, further down in the file.